src/App.asm

```diff
@@ -1,6 +1,137 @@
+# Made With Love <3
 .data
+	# ------Creating String variables In Memory-----------------
+	MainInputMessage: .asciiz "Enter total number of entries: "
+	enterNumbers: .asciiz "Enter a Number: "
+	totalPositiveMessage: .asciiz "\n\nThe Total of Positives is: "
+	totalNegativeMessage: .asciiz "   -||-   The Total of Negatives is: "
+	
+	# ------Creating Integer Variables in Memory----------------
+	max: .word 0
+	enteredNumber: .word 0 
+	countPositive: .word 0
+	countNegative: .word 0
+	i: .word 0
+	
 
 
 .globl main
 
-.text
\ No newline at end of file
+
+
+.text
+
+	main:
+		# print the MainInputMessage message to user
+		li	$v0, 4
+		la	$a0, MainInputMessage
+		syscall
+
+		# read read what the user inputed
+		li $v0, 5
+		syscall
+		move  $t0, $v0 # Move what was read to another register to use later
+		sw $t0, max # Here we used the register to move the value to memory
+
+		lw $t2, i # Loaded the i (which is 0) into register t2
+		lw $t0, max # Loaded the max (which is what the user input was) into register t0
+		lw $t4, countNegative # Loaded the the variable countNegative (which is 0) to rigster t3
+
+
+		#  We Sperated the while loop to 3 functions for cleaner code
+		
+		
+ 		# ------Start Of While Loop------------------
+ 		
+ 		while: #where the loop starts
+ 		blt	$t2, $t0, while_body # if max > i we'll jump to the body 
```

Comment:
variables (`i` and `max`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t2, i
lw  $t0, max
blt $t2, t0, while_body
...
```